catalog.tests.bom

```diff
@@ -0,0 +1,139 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+brooklyn.catalog:
+  id: brooklyn-postgresql-node-tests
+  version: 0.1
+  itemType: template
+  iconUrl: https://wiki.postgresql.org/images/a/a4/PostgreSQL_logo.3colors.svg
+  name: PostgreSQL Node tests
+  description: Test that PostgreSQL Node is running correctly
+  license_code: Apache-2.0
+  license_url: http://www.apache.org/licenses/LICENSE-2.0.txt
+  item:
+    services:
+    - type: brooklyn-postgresql-node
+      id: postgresqlnode
+      name: PostgreSQL Node
+      brooklyn.config:
+        datastore.creation.script.contents: |
+          CREATE DATABASE temporary;
+          USE temporary;
+          CREATE TABLE users (name VARCHAR(20), owner VARCHAR(20), creation_date DATE, updated_date DATE);
+          INSERT INTO users VALUES ('mysql', 'mysql', CURRENT_DATE, CURRENT_DATE)
+    - type: org.apache.brooklyn.test.framework.TestCase
+      name: PostgreSQL Node tests
+      targetId: postgresqlnode
+      brooklyn.config:
+        timeout: 15m
```

Comment:
I was thinking we'd define this as `brooklyn.parameters` so that it could be overridden easily, where required, for clouds that are slow to provision VMs etc. (see earlier comment with example yaml).